app/services/katello/pulp3/content_view_version.rb

```diff
@@ -0,0 +1,65 @@
+module Katello
+  module Pulp3
+    class ContentViewVersion
+      def initialize(smart_proxy:, content_view_version: nil)
+        @smart_proxy = smart_proxy
+        @content_view_version = content_view_version
+      end
+
+      def exporter_name
+        @content_view_version.name.gsub(/\s/, '_')
+      end
+
+      def generate_exporter_id
+        "#{@content_view_version.organization.label}_#{exporter_name}"
+      end
+
+      def generate_exporter_path
+        export_path = "#{@content_view_version.content_view}/#{@content_view_version.version}".gsub(/\s/, '_')
+        "#{@content_view_version.organization.label}/#{export_path}"
+      end
+
+      def api
+        ::Katello::Pulp3::Api::Core.new(@smart_proxy)
+      end
+
+      def repository_hrefs
+        version_hrefs.map { |href| version_href_to_repository_href(href) }.uniq
+      end
+
+      def version_hrefs
+        if @content_view_version.default?
+          @content_view_version.repositories.pluck(:version_href)
+        else
+          @content_view_version.archived_repos.pluck(:version_href)
+        end
+      end
+
+      def version_href_to_repository_href(version_href)
+        version_href.split("/")[0..-3].join("/") + "/"
+      end
+
+      def create_exporter(export_base_dir: nil)
```

Comment:
You could just set the keyword arg to Setting[:pulp_export_destination] by default!